script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -727,56 +853,68 @@ The monastery.
+ sealed off from the rest of society.
 }
 [sha:5dcfcaf28685addb3815503e5075d151b796f15d]{
 -- Page 478, Offset 19464.
 -- 修道女はその後の一生を院内で完結させる事を誓約し、この完成された共同体の一員となる。
-The nuns pledge to spend the rest of their lives inside the monastery, becoming members of this perfected collective.
+Nuns who pledge to spend the remainder of their lives inside its walls become members of this perfected collective.
 }
 [sha:7ccc13bf103c9085c4c29fbf8acb87ae28499ad3]{
 -- Page 478, Offset 19465.
 -- 彼らは外の世界と壁一枚を隔て、清貧・貞潔・従順を守りながら、父の子として相応しい一生を終える。
-Walled out from the outside world, they lead a life of poverty, purity and obedience and pass away in a manner befitting for Children of the Father.
+Cut off from the outside world,
+ they live out their days upholding the ideals of purity, obedience,
+ and freedom from material desires, as befits children of the Heavenly Father.
 }
 [sha:ffb42e4921d1fdb8cc859773eb8ea41259d4ac15]{
 -- Page 479, Offset 19466.
 -- 望んで修道院の門を叩く者はいい。
-Suitable for those who willingly knock on the doors of the monastery.
+For those who knock willingly on the doors of the monastery, it is a worthy life.
 }
 [sha:92df72fd036e6d2255dc76930bc32ad60a0bc9af]{
 -- Page 479, Offset 19467.
 -- しかしこの閉ざされた世界を望まない者にとって、修道院送りは拷問に近い。
-But for those who do not wish to be sealed in these closed worlds, being sent to a monastery is akin to torture.
+But for those who do not wish to be sealed away in this cloistered world,
+ being sent to the monastery is akin to torture.
 }
 [sha:221bc8d865f9dfefc93d5bdd4080a02185b8c0fd]{
 -- Page 480, Offset 19468.
 -- ある理由から<彼女|ノエル>には代行者以外の道が閉ざされていた。
-Due to various reasons, no other path than that of the Executor is open to Noel.
+Due to her circumstances, the only path not denied to Noel is that of an Executor.
 }
 [sha:a28bb3702b8a4814c3623494ec37851091538b7b]{
 -- Page 480, Offset 19469.
 -- なので修道院での再教育は代行者としてのものになる。若い時分なら耐えられた過酷な鍛錬に、今の自分がついていけるとは、彼女にはとても思えない。
-She will thus be reeducated at a monastery as an Executor. Though she could have been able to endure their severe training in her youth, as she is now, she certainly doesn't think it possible.
+As such, the re-education curriculum she would face at the monastery would be that of an Executor.
+ While she'd been able to endure the harsh training once before in her youth,
+ she knew there was no chance she'd be able to do so again as an adult.
 }
 [sha:a8fb06843d1618cefca7e5ea80e6131be4ebe1da]{
 -- Page 481, Offset 19470.
 -- なにより―――今、修道院に送られては身体を検査されてしまう。
-Most importantly―――If she's sent to a monastery, her body will be inspected.
+But more importantly――if she's sent to the monastery, her body will be inspected.
 }
 [sha:3e3fe843a3b4168a804b190a2d3e8d2e33bfcac6]{
 -- Page 481, Offset 19471.
 -- 代行者にとって、貞潔とは異性との交わりではなく、人体として清らかであるかの定義だ。
-To an Executor, purity doesn't refer to relations with the opposite sex but to the immaculateness of the human body.
+// Love how this is here partly just to establish that "Ciel is permitted to Fuck"
```

Comment:
legal 🙏

Reply:
tsukihime 18+ patch confirmed for pc release 2023